contracts/contracts/zapper/WOETHCCIPZapper.sol

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { IRouterClient } from "@chainlink/contracts-ccip/src/v0.8/ccip/interfaces/IRouterClient.sol";
+import { Client } from "@chainlink/contracts-ccip/src/v0.8/ccip/libraries/Client.sol";
+import { IERC20 } from "@chainlink/contracts-ccip/src/v0.8/vendor/openzeppelin-solidity/v4.8.0/contracts/token/ERC20/IERC20.sol";
+import { SafeERC20 } from "@chainlink/contracts-ccip/src/v0.8/vendor/openzeppelin-solidity/v4.8.0/contracts/token/ERC20/utils/SafeERC20.sol";
+import { IERC4626 } from "./../../lib/openzeppelin/interfaces/IERC4626.sol";
+import { IOETHZapper } from "./../interfaces/IOETHZapper.sol";
+
+/**
+ * @title WOETH CCIP Zapper Contract
+ * @notice Hanldes logic to easily zap ETH on mainnet to WOETH on L2s like Arbitrum.
+ * @author Origin Protocol Inc
+ */
+
+contract WOETHCCIPZapper {
+    using SafeERC20 for IERC20;
```

Comment:
Since we are only working with known coins with good behavior, we can drop the SafeERC20, and just use the ERC20 interface? It doesn't look like we are using safeApprove or safeTransfer anyway.